### ISSUE TITLE
Grab items from Reinforced Chests

### ISSUE THREAD

**Issue by Subzeros14** [abearcodes/Valheim#12]
Pretty much exactly what the title says. It seems that any of my items in reinforced chests do not appear in the crafting options, but if I move the items to regular chests everything works just fine.

**Comment by abearcodes** [comment]
Thanks for your report! 

- Can you check please set the setting `DebugViableContainerIndicatorEnabled`  to true and see if the container that holds these items is marked with a "+" sign? 
![image](https://user-images.githubusercontent.com/30690616/109417674-ea83f700-79c4-11eb-8e66-69ee5744ceb2.png)
- Is there any output in the BepInEx console? 
- Do you have any other mods installed?

**Comment by Subzeros14** [comment]
I do have a few other mods installed. Valheim Plus, BetterUI, Quick Stack and Deposit, Faster Teleportation, and Inventory Sort. The reinforced chests I have with items do appear to have that little purple + on them. After some quick testing, if you move the chests using Valheim Plus' advanced editing mode it appears the mod no longer recognizes the chests as being there. 

Deleting them and replacing them appears to fix this. A little inconvenient, but not too bad.

**Comment by abearcodes** [comment]
Yeah... CraftingWithContainers checks whether or not a chest was created by a player (so that you are not looting naturally spawned chests). My best guess is that Valheim Plus for w/e reason places the chest without a proper creator id. I'll test this at some point when I have the time, glad you at least found a temporary workaround

**Comment by mrei-kiel** [comment]
This seems to still be the case, is there any way to have moved chests to still be considered by CraftingWithContainers?
I have nearly all my chests moved to align them better preventing the withdrawal of items... Thank you!
